utils/_context/_scenarios/endtoend.py

```diff
@@ -389,18 +417,35 @@ def _start_interface_watchdog(self):
             [interfaces.library, interfaces.agent] + [container.interface for container in self.buddies]
         )
 
+    def _set_weblog_domain(self):
+        if self.enable_ipv6:
+            from utils import weblog  # TODO better interface
+
+            if sys.platform == "linux":
+                # on Linux, with ipv6 mode, we can't use localhost anymore for a reason I ignore
+                # To fix, we use the container ipv4 address as weblog doamin, as it's accessible from host
+
+                weblog.domain = self.weblog_container.network_ip(self._network)
+            elif sys.platform == "darwin":
+                # on Mac, this ipv4 address can't be used, because container IP are not accessible from host
+                # as they are on an network intermal to the docker VM. But we can still use localhost.
+                ...
```

Comment:
add a logger.info?

Reply:
addressed in [3aae93a](https://github.com/DataDog/system-tests/pull/3624/commits/3aae93ab6692861a39a315dea348460a3119bb2b)